metadata-models/src/main/pegasus/com/linkedin/common/TagAssociation.pdl

```diff
@@ -0,0 +1,12 @@
+namespace com.linkedin.common
+
+/**
+ * Properties of an applied tag. For now, just an Urn. In the future we can extend this with other properties, e.g.
+ * propagation parameters.
+ */
+record TagAssociation {
```

Comment:
What is the difference between a Tag and a TagAssociation?

Reply:
TagAssociation gives us the flexiblity to apply ownership, access control and other metadata to a tag application